src/init/features/functions/typescript.js

```diff
@@ -38,13 +39,13 @@ module.exports = function (setup, config) {
           'npm --prefix "$RESOURCE_DIR" run build',
         ]);
         return config
-          .askWriteProjectFile("functions/package.json", PACKAGE_LINTING_TEMPLATE)
+          .askWriteProjectFile("functions/package.json", getPackage(setup.functions.lint))
```

Comment:
Since we know `setup.functions.lint` is `true` at this point, let's explicitly pass it.

Reply:
Alright, I'll also use boolean value for the other call of `getPackage()`.